packages/plugin-full-text-search/src/server/middlewares/search.ts

```
@@ -0,0 +1,88 @@
+    sqlite: () => new FieldSqlite(),
+  };
+
+  const handler = handlers[dbType];
+  if (!handler) {
+    throw new Error(`Unsupported database type: ${dbType}`);
+  }
+  return handler();
+}
+
+export async function searchMiddleware(ctx: Context, next: Function) {
+  const params = ctx.action.params;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)